### PR TITLE
Add r-ladies about page

### PR DESCRIPTION
**Pull request by sophie-a-lee**

### CODE REVIEW

File: rladies.qmd [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  template: jolla
+  links:
+    - icon: twitter-x
+      text: Twitter
+      href: https://twitter.com/RLadiesSheff
+---
+
+::: {style="text-align: left;"}
+
+We are Sheffield R-ladies, a chapter of [R-Ladies Global](https://rladies.org/). The mission of R-Ladies is to achieve proportionate representation by encouraging, inspiring, and empowering people of genders currently under-represented in the R community. R-Ladies’ primary focus, therefore, is on supporting minority gender R enthusiasts to achieve their programming potential, by building a collaborative global network of R leaders, mentors, learners, and developers to facilitate individual and collective progress worldwide.
```
Comment (ns-rse) [inline rladies.qmd]: This and the following paragraph are failing the [markdownlint-cli2](https://github.com/DavidAnson/markdownlint/) checks performed by [pre-commit](https://pre-commit.com).

Lines need hard-wrapping at 120 characters (from memory RStudio soft-wraps lines).

**Review by ns-rse — CHANGES_REQUESTED** [review]
Thanks for the PR @sophie-a-lee :+1: 

The pre-commit checks are currently failing but should be trivial to fix (a few carriage returns will fix it).

If you don't have pre-commit configured locally and would like to learn more I've a few [blog posts on using it](https://blog.nshephard.dev/#category=pre-commit) that might be useful. Start with the oldest first, it focuses on Python but covers the principles, there is a more recent one on R. Give me a shout if you have any questions or want some help.

**Comment by ns-rse** [comment]
Hi @sophie-a-lee 

It looks like you accidentally force-pushed something that has undone the changes you made. Perhaps you synchronised your fork with upstream. Not sure.

Two options going forward though...

1) You could make a branch on your local copy of your fork, add the changes (and hard wrap lines at 120 characters so that the `pre-commit.ci` changes pass) and make a new pull request.
2) I could make the changes based on [this commit](https://github.com/SheffieldR/SheffieldR.github.io/commit/3fabf445114fea0993d746528a1b047ad18e2b62).

Happy with either, let me know which you would like to do.

@ns-rse

**Comment by ns-rse** [comment]
Had some time in my lunch break and have added the content via #57 . I hope that is ok @sophie-a-lee .

**Comment by sophie-a-lee** [comment]
Brilliant! Thanks, Neil. Sorry, I was planning to try sorting it earlier but got dragged onto something else!

**Pull request closed without merging**